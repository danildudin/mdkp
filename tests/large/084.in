50 19 0
276 427 112 203 938 115 268 79 682 386 409 654 567 993 800 123 298 224 729 512 152 809 477 938 317 30 877 414 141 179 897 586 845 989 879 213 233 991 812 270 889 237 229 248 291 775 2 694 142 351 
847 15 61 588 873 39 876 933 895 290 616 134 841 883 140 321 129 325 556 311 367 643 970 490 674 250 519 448 185 507 449 608 765 641 688 474 514 576 719 321 64 185 453 466 19 822 35 369 788 18 
30 208 475 557 183 383 456 755 5 873 882 906 707 833 617 459 929 723 603 925 826 720 780 755 353 98 164 288 254 501 772 502 340 1000 837 440 579 833 633 960 50 331 537 255 300 423 337 478 93 518 
555 711 536 99 891 68 668 545 609 298 43 317 227 390 27 283 135 825 45 293 814 136 703 983 723 612 507 770 954 943 504 988 609 619 532 510 238 764 394 325 997 242 34 529 413 795 859 812 581 142 
844 126 495 390 47 341 863 790 587 604 211 676 478 563 666 52 814 322 194 356 323 197 577 397 46 504 467 327 866 708 488 992 116 839 204 897 848 166 5 837 519 252 163 947 974 308 539 666 329 745 
644 685 388 303 475 41 232 455 63 158 359 247 188 936 721 974 38 871 710 882 890 659 250 807 909 984 798 661 842 708 210 184 386 993 140 609 929 411 621 200 922 760 257 65 999 383 141 49 939 200 
705 953 494 858 902 297 840 266 673 471 7 770 494 676 683 638 970 946 218 396 129 240 689 841 680 485 496 158 749 142 594 424 842 52 36 94 435 978 119 580 7 937 287 223 205 850 779 332 162 331 
856 603 61 36 676 262 529 713 859 507 292 164 292 197 934 605 993 274 694 540 534 285 448 337 267 712 884 466 747 510 54 512 813 476 273 354 203 410 359 174 279 685 554 794 231 455 656 727 329 893 
250 76 108 346 983 648 507 620 114 978 10 536 505 43 304 3 250 796 621 840 767 339 400 782 600 274 458 864 244 585 401 381 721 956 956 609 605 269 53 394 839 395 149 228 672 944 530 693 285 649 
621 277 645 620 415 680 422 297 441 713 877 587 989 165 848 767 828 828 718 397 83 841 140 218 545 853 570 16 772 814 694 368 182 101 5 808 55 69 808 517 902 486 367 3 810 913 580 754 269 980 
870 149 307 500 326 425 356 922 313 622 17 927 442 240 25 368 555 540 327 644 604 624 304 945 22 212 538 686 373 16 783 326 958 351 746 23 337 721 690 938 481 915 540 709 678 825 732 161 706 580 
308 582 112 171 944 477 886 997 563 439 809 190 507 678 460 204 111 508 743 63 857 46 463 600 805 7 645 717 467 983 180 122 545 742 279 158 155 618 425 94 863 341 767 47 327 233 775 265 973 220 
806 560 965 907 80 727 983 825 982 193 218 680 779 408 359 532 204 613 799 636 709 912 144 649 902 600 71 889 115 43 771 565 648 270 767 776 754 491 846 214 779 807 231 471 107 357 535 623 457 496 
56 580 756 908 64 3 393 946 128 661 148 552 94 679 384 323 907 199 651 310 246 419 79 668 310 945 753 590 159 256 524 388 59 547 276 276 521 2 157 645 303 937 18 788 114 254 336 321 675 515 
725 654 576 993 417 845 933 376 465 984 950 510 964 304 164 692 648 336 412 932 91 895 615 792 571 67 167 807 450 583 980 914 144 301 135 687 698 804 242 172 762 354 470 787 287 489 17 615 264 724 
829 216 674 266 629 676 967 668 742 661 740 663 960 774 713 796 693 628 979 38 157 985 808 609 944 947 444 350 53 240 231 873 378 181 996 830 851 657 680 543 203 655 587 106 446 778 711 914 48 183 
315 75 629 851 881 719 686 558 857 432 767 720 335 926 791 160 661 142 164 726 796 556 461 532 198 109 256 660 168 86 188 574 278 100 357 341 700 413 777 38 388 574 483 926 441 802 332 811 936 167 
534 688 937 764 646 450 939 753 66 888 622 99 357 342 637 998 847 1000 456 163 252 332 760 323 526 951 446 209 616 489 334 843 929 926 733 983 684 895 160 179 97 650 966 180 943 135 733 602 364 192 
739 624 782 651 963 252 519 959 740 123 977 105 75 253 367 485 487 439 650 928 539 486 276 31 541 720 704 814 137 773 786 822 93 134 620 98 773 610 535 843 909 26 706 921 472 472 766 852 891 3 
976 689 118 97 595 24 567 727 98 323 83 900 852 796 144 239 740 692 683 298 12 739 325 929 547 145 221 902 464 456 906 960 308 762 401 214 17 68 764 953 617 423 750 424 68 797 782 872 406 200 
63498 6805 59529 52089 90100 13582 59068 41027 16758 53831 18259 10076 57597 14900 87085 37847 38509 33897 4990 
