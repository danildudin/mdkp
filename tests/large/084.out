10414
14
4 8 10 13 14 26 32 34 36 37 38 40 44 45 
